src/main/io/ledstrip.c

```diff
@@ -862,13 +995,13 @@ void updateLedStrip(void)
 
     // test all led timers, setting corresponding bits
     uint32_t timActive = 0;
-    for(timId_e timId = 0; timId < timTimerCount; timId++) {
-        if(cmp32(now, timerVal[timId]) < 0)
+    for (timId_e timId = 0; timId < timTimerCount; timId++) {
+        if (cmp32(now, timerVal[timId]) < 0)
```

Comment:
if ((cmp32(now, timerVal[timId]) < 0) && (cmp32(now, timerVal[timId]) > -LED_STRIP_MS(5000)))

limit timer delay to 5s (sanitize later)